Dockerfile

```
@@ -1,20 +1,31 @@
 # vim: set expandtab sw=4
-FROM golang:1.22.7-bullseye
+FROM golang:1.22.7-bookworm
 
 # Create a non-priviledged user to run browsers as (Firefox and Chrome do not
 # like to run as root).
 RUN chmod a+rx $HOME && useradd --uid 9999 --user-group --create-home browser
 
+# Add apt repositories for Java, Node.js and Google Cloud CLI
+RUN export DISTRO_CODENAME=$(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) && \
+    echo "deb [signed-by=/usr/share/keyrings/correto.gpg] https://apt.corretto.aws stable main" > /etc/apt/sources.list.d/adoptium.list && \
```

Nit: I see adoptium here. I don't see it in the [instructions](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/generic-linux-install.html#debian-install-instruct). Let's change it to correcto
```suggestion
    echo "deb [signed-by=/usr/share/keyrings/correto.gpg] https://apt.corretto.aws stable main" > /etc/apt/sources.list.d/corretto.list && \
```

---

Oops, good catch. I also fixed the typo in the keyring file name.